helmfile/charts/starboard/ciskubebench-exporter/templates/rbac.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ciskubebench-exporter
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ciskubebench-exporter
+rules:
+  - apiGroups: ["aquasecurity.github.io"]
+    resources: ["ciskubebenchreports"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+    - extensions
+    resourceNames:
+    - privileged
+    resources:
+    - podsecuritypolicies
```

Comment:
Do we need the full privileged psp? Otherwise if you have time you could create a more limited one.

Reply:
Unsure, i just copied the vulnerability-exporter. But i can check.

Reply:
It requires to run as root, as it is using [cron](https://github.com/elastisys/compliantkubernetes-apps/pull/1122/files/cdbad6a6e72ac074cd9867bac3414255b4adf2bf#diff-984bfadac9edbc1b562a0e66213baf6f9b8a350f420d43e4b3fcdc8bd185b4c7R9). I can create an image that adds a group for cron which which our user is part of.

Reply:
Skipping for now.